static/js/admin/performance_visualizations/assets/ramda/unc-custom-build.js

```diff
@@ -0,0 +1,162 @@
+//  Ramda v0.21.0
```

Comment:
It is a build of just the bits of ramda.js used, instead of the whole library. Might be better renamed to something like ramda-merge-with-key.js, which describes what's in it.